natgeo/mortar
# [SYTO-1140] Fixes issues with Most Read/Latest components

https://jira.natgeo.com/browse/SYTO-1140
https://jira.natgeo.com/browse/SYTO-1145

These changes fix bug where promos would duplicate until there were 4 promos in the four up promo. 
Also included is a fix for the EQ utility, to properly scope the CSS classes to that PodPromo doesn't take over the wrapping class divs
And a bonus change for fixing the wrapping CTA on Firefox at the mobile breakpoint in the POD promo

To test:

- Pull SYTO-1140
- `npm test`
- `gulp serve`
- verify latest/most read are getting wrapped with the proper classNames via the EQ utility at tablet and desktop 
- verify POD promo CTA at the mobile breakpoint
- verify POD promo has proper wrapping class at various breakpoints
